src/displaz.jl

```diff
@@ -25,7 +25,19 @@ end
             vertices[2*i-1] = ls[i][0.0]
             vertices[2*i] = ls[i][end]
         end
-        Displaz.plot3d(vertices, args...; label = label, markershape = '-', linebreak = 2, kwargs...)
+        # Allow colors to be assigned to vectors of lines
+        if haskey(kwargs, :color) && typeof(kwargs[:color]) <: Array{N,2} where N <: Number
+            colour = repeat(kwargs[:color],inner=[1, 2])
+            Displaz.plot3d(vertices, args...; label = label, markershape = '-', linebreak = 2, kwargs..., color=colour)
+        elseif haskey(kwargs, :color) && typeof(kwargs[:color]) <: Array{String}
+            colour = cat(([repeat(Displaz._color_names[c[1]],outer=[1,2]) for c in kwargs[:color]])..., dims=2)
```

Comment:
just in case if you want to know, this will give the same result

`reduce(hcat, [repeat(Displaz._color_names[c[1]],outer=[1,2]) for c in kwargs[:color]])`